Makefile.am

```diff
@@ -277,6 +277,7 @@ AM_CPPFLAGS = \
 	-I$(top_srcdir)/src \
 	-DSRCDIR=\"$(abs_srcdir)\" \
 	-DBUILDDIR=\"$(abs_builddir)\" \
+	-DLIBEXECDIR=\"$(libexecdir)\" \
```

Comment:
That's exactly the same as the existing `-DPACKAGE_LIBEXEC_DIR` two lines down. IMHO the latter should be renamed, and adjusting src/ws/cockpitauth.c, but let's please not use both.

Reply:
Good catch!!